src/app/public/modules/href/types/href-parameters.ts

```diff
@@ -0,0 +1 @@
+export type SkyHrefParameters = { [_: string]: string };
```

Comment:
Should this be exported to the public API?

Reply:
After talking with Paul, the approach is going to be to put parameters into the `skyHref` attribute more akin to how Angular handles parameters, so this type is no longer needed.